### ISSUE TITLE
Visualize time-series data in Workspace

### ISSUE THREAD

**Issue by Zodiase** [openskope/skope-interface#105]
Visualize time-series data of a single variable in charts.
The user should be able to select a point on the map and a time range from the entire time range of the dataset.

**_For now we don't need to consider performance requirements._**

The request sent to backend for the data would contain 3 pieces of information: spatial constrain, temporal constrain and variable name.

The response should contain some metadata about the payload (for future extensions) and a list of data points of the time-series to be rendered. Each data point should also contain some uncertainty value that should also be rendered.

Things we need to finalize before implementation:
- [x] Request schema. How to encode all 3 pieces of info in the request? We might want the schema to be flexible enough for future extensions (e.g. specify a polygon in the spatial constrain.)
- [ ] Response schema. What info does the client side need for the charting and other related stuff?

**Comment by Zodiase** [comment]
C3 Reference: http://c3js.org/reference.html

C3 Zoom Chart: http://c3js.org/samples/interaction_zoom.html

C3 Sub Chart: http://c3js.org/samples/options_subchart.html

C3 Area Chart: http://c3js.org/samples/chart_area.html
Can the area be not zero-based?

C3 Timeseries Chart: http://c3js.org/samples/timeseries.html

**Comment by Zodiase** [comment]
Proposed request schema:
```
POST
Content-Type: application/json; charset=utf-8

{
    datasetId: <string>,
    variableName: <string>,
    boundaryGeometry: {
        type: <string> ("Point" | "Polygon"),
        coordinates: <Array>,
    },
    period: {
        gte: <Date>,
        lte: <Date>,
    },
}
```

**Comment by Zodiase** [comment]
Current UI as of 4b1a65c03393d39b8be223f2bd22ae0006e6cced
![image](https://user-images.githubusercontent.com/1815174/36596004-b926f60e-1850-11e8-8e8b-a28847526445.png)

**Comment by Zodiase** [comment]
Rest of the work is addressed in #112, #113 and #114.
